packages/type-system/rust/tests/web.rs

```diff
@@ -0,0 +1,10 @@
+//! Test suite for the Web and headless browsers.
+
+#![cfg(target_arch = "wasm32")]
+
+extern crate wasm_bindgen_test;
+use wasm_bindgen_test::*;
+
+wasm_bindgen_test_configure!(run_in_browser);
+
+// TODO: Decide which tests to write here
```

Comment:
So `wasm-pack` has this nice feature where it will automatically download a driver for a browser on your system and will compile the module into WASM and actually run these tests (written normally in Rust) inside your browser. That seems quite helpful but I think it basically runs it inside WASM, so it's not testing the bindings, which is why I ended up going with the Jest setup. Perhaps there are still some tests we'd like to add here though.